vite.config.ts

```diff
@@ -63,6 +63,15 @@ export default defineConfig(({ mode }) => {
   // Allow hot reloading of json files in public folder when in development
   if (env.NODE_ENV === 'development') plugins.push(jsonHMR());
 
+  if (env.VITE_APP_INCLUDE_CODECOV === 'true')
```

Comment:
Missing the import above, not sure why it didn't complain but its in the example.
```
import { codecovVitePlugin } from "@codecov/vite-plugin";
```